### ISSUE TITLE
[Forms] Review all form elements standards and accessibility 

### ISSUE THREAD

**Issue by atomworks** [future-standard/scorer-ui-kit#124]
We should audit all of our form components to make sure that we're following best practices to make sure standard interaction patterns are all available and working. This means being able to tab between form fields; making sure we can pass tab ordering to components as needed; that number fields are able to use the arrow keys; etc.

Although we are not pushing for full accessibility within this task we should be covering the fundamentals as part of making sure standards are met. A quick review using the screenreader feature on the iPhone (for example) would be a worthwhile step.

This review should include making sure that the `Slider` component can be interacted with using tabs and arrow keys.

_From original BugHerd comments:_
>This thought came up as we were discussing the use of the for attribute on elements and if they're required or optional.

> On the whole we should look at this in more detail along with the basic semantics and accessibility to make sure we're doing it right.

> Additionally, elements like sliders and other non-standard controls should be accessible when tabbed too and with the use of keyboard too.

**Comment by atomworks** [comment]
Moved to project management
